vscode/oso/Makefile

```diff
@@ -1,20 +1,28 @@
-.PHONY: build wasm fmtcheck lint test
+.PHONY: build package wasm fmtcheck lint test clean
 
-build: wasm node_modules
+build: clean wasm node_modules
 	@yarn compile
 
+package: clean wasm node_modules
+	@yarn vsce package
```

Comment:
I don't think you need to use `@yarn` here (and elsewhere). It's intentional that `make` prints out the command being executed so you can understand what's going on for inspection reasons. Usually `@` is only used to silence logging commands like `@echo "Running"`.

See: https://www.gnu.org/software/make/manual/html_node/Echoing.html#Echoing

Reply:
Ah okay that's good info — I've been adding it everywhere b/c I didn't find any value from printing the commands out. Will remove

Reply:
d34bb70a